### PR TITLE
[AUTOTVM] API change

### PR DESCRIPTION
**Pull request by merrymercy**
Make API more consistent and clear.

in measure_option:
* parallel_num -> n_parallel
* autotvm.use_rpc -> autotvm.measure.rpc

### CODE REVIEW

**Comment by merrymercy** [comment]
cc @Jokeren @FrozenGene @feng98

File: tutorials/autotvm/tune_nnvm_arm.py [hunk 1]
```diff
@@ -191,9 +191,9 @@ def get_network(name, batch_size):
    'early_stopping': 250,
 
    'measure_option': autotvm.measure_option(
-       autotvm.use_rpc(device_key, host='localhost', port=9190),
+       autotvm.measure.rpc(device_key, host='localhost', port=9190),
        number=4,
-       parallel_num=1,
+       n_parallel=1,
```
Comment (merrymercy) [inline tutorials/autotvm/tune_nnvm_arm.py]: cc @Jokeren @FrozenGene @feng98

**Comment by Jokeren** [comment]
Thanks for notifying!

**Comment by FrozenGene** [comment]
Thanks.

**Comment by feng98** [comment]
Thanks.

**Pull request merged**